frontend/src/components/details/person.svelte

```diff
@@ -1,14 +1,16 @@
 <script lang="ts">
-  import { mediaIdToUrlConverter } from "../../utils"
+  import { mediaIdToUrlConverter, calculateAmountOfShownItems } from "../../utils"
   import type { Cast } from "../../types"
 
   export let cast: Cast[]
 
   const LOW_RES_IMG_URL = "https://image.tmdb.org/t/p/w500/"
-  const SHOW_BUTTON_AMOUNT = 18
-  const CAST_ITEM_START_AMOUNT = 9
 
-  let castItemAmount = 9
+  let castItemAmount = calculateAmountOfShownItems(
+    window.visualViewport.width,
+    [27, 24, 21, 15, 12, 9]
+  )
```

Comment:
Would be sick with an interface for the function or at least an in-line comment to explain what the numbers are for

Something like:
```ts
interface ViewportAmount {
  2xl: number
  xl: number
  ect...
  ```

Reply:
I wrote some more stuff about this on `calculateAmountOfShownItems()`

Reply:
Done